src/garage/tf/models/gaussian_conv_model.py

```diff
@@ -0,0 +1,301 @@
+"""GaussianConvModel."""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.core.cnn import cnn
+from garage.tf.core.mlp import mlp
+from garage.tf.core.parameter import parameter
+from garage.tf.distributions import DiagonalGaussian
+from garage.tf.models.base import Model
+
+
+class GaussianConvModel(Model):
+    """
+    GaussianConvModel.
+
+    Args:
+        filter_dims(tuple[int]): Dimension of the filters. For example,
+            (3, 5) means there are two convolutional layers. The filter
```

Comment:
Can I have 2 filters which are (5x7) followed by (3x3)? why not? how would I specify that? `((5, 7), (3, 3))`?

Reply:
I don't think the implementation of CNN in garage supports filters of different dimensions like `(5,7)`.

Reply:
It doesn't but I agree we should have support for that. I suggest filing an issue and making that as a feature to be implemented later as that's out of the scope of this PR.

Reply:
Yes, will file an issue.